src/FakeItEasy/Core/FakeManager.PropertySetterRule.cs

```diff
@@ -32,17 +32,34 @@ public void Apply(IInterceptedFakeObjectCall fakeObjectCall)
             {
                 Guard.AgainstNull(fakeObjectCall, nameof(fakeObjectCall));
 
-                var newRule = new CallRuleMetadata
-                                  {
-                                      CalledNumberOfTimes = 1,
-                                      Rule = new PropertyBehaviorRule(fakeObjectCall.Method, this.fakeManager)
-                                                 {
-                                                     Indices = fakeObjectCall.Arguments.Take(fakeObjectCall.Arguments.Count - 1).ToArray(),
-                                                     Value = fakeObjectCall.Arguments.Last()
-                                                 }
-                                  };
-
-                this.fakeManager.AllUserRules.AddFirst(newRule);
+                //// Setting the property adds a PropertyBehaviorRule to store the assigned value
+                //// The PropertyBehaviorRule is added at the end of user rules to avoid overriding
+                //// explicit configurations of the property made with CallTo.

```

Comment:
The comment applies to the next statement (the LINQ query).

I'm a bit confused... I *think* I had introduced the blank line and the `////` to make StyleCop happy, but there's no warning with `//` and no blank line, so I'll do just that.

Also, it seems I had missed @blairconrad's comment...